### DISCUSSION TITLE
Please add Btools (maybe F3X, it won't make a difference)

### DISCUSSION THREAD

**Discussion by TheHackerLxr** [kohls-admin/kohls-admin#19] [Ideas]
it's convenient to change the map in real time, many use it in different ways, some for trolling, and place creators can use it for events :)
Thanks so much!

**Comment by kohltastrophe** [comment]
I don't intend to include F3X by default with this new system, however I will include default Roblox building tools and provide an example on how to easily use F3X or any tool from `game.ServerStorage` via the `give` command.

**Reply by TheHackerLxr** [reply]
Yeah that would be still cool!
